### ISSUE TITLE
[Bug]: Received Ruins Greatsword without boss fight

### ISSUE THREAD

**Issue by zzhengnan** [LukeYui/EldenRingSeamlessCoopRelease#425]
### Discord Username

sturgeonBobby#1475

### Describe the issue in detail

Thanks for this great mod and your hard work!

I was teleported to the Redmane Castle from the Impassable Greatbridge. At this point, my friend joined my session as a wanderer, upon which
* We both received the [Ruins Greatsword](https://eldenring.wiki.fextralife.com/ruins+greatsword) _**without**_ needing to fight the area bosses in the Redmane Castle Plaza -- Misbegotten Warrior, and Crucible Knight
* The two bosses appeared to have been defeated _**permanently**_. The plaza was no longer gated by yellow mist and we couldn't go back and fight the bosses again

Some additional context
* Neither of us had been to the Redmane Castle or fought the bosses before
* Neither of us had started the Radahn quest line. We were simply exploring the map when the bug happened

On a separate note, is there a way to modify our save files so that we can go back and fight the bosses again?

### Steps to recreate the bug

Not sure if it's reproducible but the following sequences triggered the bug in our case
* Host to teleport to Redmane Castle from the Impassable Greatbridge
* Wanderer to join the session

### Version

1.3.0 (Latest)

**Comment by Yoni0040** [comment]
Hey man, in single player i've beat the game twice and as far as I can tell there is a way to skip these boss battle.
Basically the castle is abandoned- no guard no nothing. Jerren is there to initiate the Radagn fight though. That's what happened in my first playthrough.
In my second playthrough however there were guards and everything.
What changed?
well I think it has something to do with ranni's questline- in my second playthrough i started her questline before reaching redmane's castle and in my opinion one of you might've started her quest.

basically if one of you starts her quests they should spawn, let's say the wanderer did. then they're alive in his world BUT as soon as he enters your world they cease to exist as if you've defeated them.

Also this is just my theory and I could be completely wrong and probably wasted both of our time.

**Comment by zzhengnan** [comment]
Thanks for the reply! My friend and I had both started Ranni's quest line, although we noticed different dialogue options when we talked to her. Our hypothesis at this point is that our save files may have de-sync'd.

**Comment by LukeYui** [comment]
I'm pretty sure this is normal game behaviour if Blaidd clears the castle before you get there

**Comment by Yoni0040** [comment]
Bruh, this has nothing to do with blaidd since I haven't even rescued him from the forest - hell i havent even interacted with him and the castle is cleared.
